updater/spec/dependabot/updater_spec.rb

```diff
@@ -2441,10 +2441,10 @@ def expect_update_checker_with_ignored_versions(versions)
 
         expect(Dependabot.logger).
           to have_received(:info).
-          with("<job_1>   version-update:semver-patch - from .github/dependabot.yaml")
+          with("  version-update:semver-patch - from .github/dependabot.yaml")
         expect(Dependabot.logger).
           to have_received(:info).
-          with("<job_1>   version-update:semver-minor - from .github/dependabot.yaml")
+          with("  version-update:semver-minor - from .github/dependabot.yaml")
```

Comment:
I believe this double-spacing is working as intended as the intent is to tab the content in from the right margin, but I'll check

Reply:
[Yep](https://github.com/dependabot/dependabot-core/blob/a60d501d79965fe739b039a598cea9b3ddc74376/updater/lib/dependabot/updater.rb#L546)

Reply:
I think you should be able to match with regex - instead of `with("  some message")` you could use `with(/some message/)` to ignore whitespace

Can't tell if capturing whitespace is intentional here